test/k8sT/manifests/kafka-sw-app.yaml

```diff
@@ -28,6 +28,17 @@ spec:
           value: "20000"
         - name: KAFKA_ZOOKEEPER_CONNECTION_TIMEOUT_MS
           value: "20000"
+        livenessProbe:
+          tcpSocket:
+            port: 9092
+          initialDelaySeconds: 30
+          failureThreshold: 10
+          periodSeconds: 10
+        readinessProbe:
+          tcpSocket:
+            port: 9092
+          initialDelaySeconds: 5
+          periodSeconds: 5
       nodeSelector:
```

Comment:
I think we want to truly randomize it we need to remove the constaints on `nodeSelector:` of all the app pods